builder/Cargo.toml

```diff
@@ -8,7 +8,7 @@ license.workspace = true
 
 [features]
 default = []
-crashtracker = ["datadog-profiling-ffi?/crashtracker-receiver", "datadog-profiling-ffi?/crashtracker-collector"]
+crashtracker = ["datadog-profiling-ffi?/crashtracker-receiver", "datadog-profiling-ffi?/crashtracker-collector", "datadog-profiling-ffi?/demangler"]
```

Comment:
Not sure whether this also has to be crashtracker-demangler instead of just demangler.

Reply:
I followed what's in the `profiling-ffi/Cargo.toml`